pilot/pkg/networking/core/v1alpha3/route/route.go

```diff
@@ -451,14 +476,16 @@ func (b SortHeaderValueOption) Swap(i, j int) {
 }
 
 // translateAppendHeaders translates headers
-func translateAppendHeaders(headers map[string]string) []*core.HeaderValueOption {
+func translateAppendHeaders(headers map[string]string, appendFlag bool) []*core.HeaderValueOption {
 	headerValueOptionList := make([]*core.HeaderValueOption, 0, len(headers))
+	appendValue := &types.BoolValue{Value: appendFlag}
 	for key, value := range headers {
 		headerValueOptionList = append(headerValueOptionList, &core.HeaderValueOption{
 			Header: &core.HeaderValue{
 				Key:   key,
 				Value: value,
 			},
+			Append: appendValue,
```

Comment:
was this option introduced in the last couple of months? If so, you need a proxy version guard here for backward compatibility.. See other code in this file for examples..

Reply:
As far as I can tell it was added in Envoy 1.5.0 with the v2 API: https://www.envoyproxy.io/docs/envoy/v1.5.0/api-v2/base.proto#envoy-api-msg-headervalueoption
This was a year ago, long before Istio 1.0. Do we still need to support it?